src/components/common/AppHeader.vue

```diff
@@ -103,6 +103,8 @@ export default {
       noScroll.off()
     },
     show() {
+      this.session.currrentModalOpen.close()
+      this.$store.commit(`setCurrrentModalOpen`, false)
```

Comment:
```suggestion
```

this is already implied by `close`

Reply:
cool. I also noticed a mistake there. Missing the conditional